src/core/readyException.js

```diff
@@ -0,0 +1,13 @@
+define( [
+	"../core"
+], function( jQuery ) {
+
+"use strict";
+
+jQuery.readyException = function( error ) {
+	window.setTimeout( function() {
+		throw error;
+	} );
```

Comment:
@gibson042 I think that could work only if we deprecated/removed `$(fn)` but this is such a popular pattern that I don't see it going away any time soon. And this fact alone is enough for me to want it not silencing exceptions.